src/System.Security.Cryptography.Algorithms/tests/RijndaelTests.cs

```diff
@@ -52,6 +52,42 @@ private static void VerifyDefaults(Rijndael alg)
         }
 
         [Fact]
+        public static void VerifyBlocksizeIVNulling()
+        {
+            using (var testIVAlg = Rijndael.Create())
+            {
+                using (var alg = Rijndael.Create())
+                {
+                    alg.IV = testIVAlg.IV;
+                    alg.BlockSize = 128;
+                    Assert.Equal(testIVAlg.IV, alg.IV);
+                }
+
+                using (var alg = new RijndaelManaged())
+                {
+                    alg.IV = testIVAlg.IV;
+                    alg.BlockSize = 128;
+                    Assert.Equal(testIVAlg.IV, alg.IV);
+                }
+
+                using (var alg = new RijndaelLegalSizesBreaker())
+                {
+                    // This one should set IV to null on setting BlockSize since there is only one valid BlockSize
+                    alg.IV = testIVAlg.IV;
+                    alg.BlockSize = 1;
+                    Assert.Throws<NotImplementedException>(() => alg.IV);
```

Comment:
The comment says that it sets IV to null, but the test says it throws a NotImplementedException...

Reply:
This is because the test class being used overrides `GenerateIV` with a `NotImplementedException` and `GenerateIV` is called whenever `get_IV` is called when the IV is currently null.

It's a bit roundabout, but I didn't want to create a new dummy test class...